ui/hooks/experiences/useRamps/useRamps.test.tsx

```diff
@@ -0,0 +1,349 @@
+import { cloneDeep } from 'lodash';
```

Comment:
i think we can change the file type to `test.ts` instead of `test.tsx`, wdyt?

Reply:
Indeed, since we are using `renderHookWithProvider` as helper. 
Is there a benefit? If we ever change the test to render inline JSX we will need it back, is it worth it to change it?

Reply:
Changed in [5335b9d](https://github.com/MetaMask/metamask-extension/pull/18862/commits/5335b9d03bf7be52f036deba55b72cc1bdf38a40)